client/src/main/java/io/mantisrx/server/master/client/HttpUtility.java

```diff
@@ -75,7 +76,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                                         super.channelRead(ctx, msg);
                                     }
                                 });
+
+                                int maxContentLength = 10 * 1024 * 1024; // Ten megabytes
+                                pipeline.replace(HttpObjectAggregator.class, "http-object-aggregator", new HttpObjectAggregator(maxContentLength));
```

Comment:
Where does `HttpObjectAggregator` get added to the pipeline? Checking to make sure `replace()` doesn't throw a `NoSuchElementException`.

Reply:
Good call, added some exception handling.